src/library.js

```diff
@@ -3201,11 +3199,164 @@ mergeInto(LibraryManager.library, {
 #if MINIMAL_RUNTIME
     var f = dynCalls[sig];
 #else
+    if (!('dynCall_' + sig in Module)) {
+      Module['dynCall_' + sig] = createDyncallWrapper(sig);
+    }
     var f = Module['dynCall_' + sig];
 #endif
     return args && args.length ? f.apply(null, [ptr].concat(args)) : f.call(null, ptr);
   },
+#if MINIMAL_RUNTIME
   $dynCall__deps: ['$dynCallLegacy', '$getWasmTableEntry'],
+#else
+  $dynCall__deps: ['$dynCallLegacy', '$getWasmTableEntry', '$createDyncallWrapper'],
+#endif
+#endif
+#if DYNCALLS || !WASM_BIGINT
```

Comment:
This is the same condition as the one about that guards `dynCallLegacy`.. can the be declared inside the same block?